<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>382</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Equation specifier</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>282</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
qproperty-alignment: 'AlignVCenter | AlignRight';
border: 1px solid gray;
background-color : white;
}


</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonMultiply">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #CEE891;
color: black; 
border: 1px solid gray;
}

QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E1F5C4, stop: 1 #E1F5C4);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonEight">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonNine">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonSeven">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonMinus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #CEE891;
color: black; 
border: 1px solid gray;
}

QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E1F5C4, stop: 1 #E1F5C4);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonFive">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonFour">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonSix">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonTwo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonPlus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #CEE891;
color: black; 
border: 1px solid gray;
}

QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E1F5C4, stop: 1 #E1F5C4);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonOne">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonThree">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonOK">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #D1E751;
color: black; 
border: 1px solid gray;
}

QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E1F5C4, stop: 1 #E1F5C4);
}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonZero">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonErase">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #DCE3E2;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonDivide">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #CEE891;
color: black; 
border: 1px solid gray;
}

QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E1F5C4, stop: 1 #E1F5C4);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonParenthesisBegin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #DCE3E2;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonClear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #DCE3E2;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonParenthesisEnd">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #DCE3E2;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>282</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
qproperty-alignment: 'AlignVCenter | AlignRight';
border: 1px solid gray;
background-color : #F8F8F8;
}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonPoint">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>320</y>
      <width>72</width>
      <height>62</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
background-color: #E9E9E9;
color: black;
border: 1px solid gray;
}
QToolButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #fcfcfc);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <zorder>display</zorder>
   <zorder>toolButtonMultiply</zorder>
   <zorder>toolButtonEight</zorder>
   <zorder>toolButtonNine</zorder>
   <zorder>toolButtonSeven</zorder>
   <zorder>toolButtonMinus</zorder>
   <zorder>toolButtonFive</zorder>
   <zorder>toolButtonFour</zorder>
   <zorder>toolButtonSix</zorder>
   <zorder>toolButtonTwo</zorder>
   <zorder>toolButtonPlus</zorder>
   <zorder>toolButtonOne</zorder>
   <zorder>toolButtonThree</zorder>
   <zorder>toolButtonOK</zorder>
   <zorder>toolButtonZero</zorder>
   <zorder>toolButtonDivide</zorder>
   <zorder>toolButtonParenthesisBegin</zorder>
   <zorder>toolButtonClear</zorder>
   <zorder>toolButtonParenthesisEnd</zorder>
   <zorder>result</zorder>
   <zorder>toolButtonPoint</zorder>
   <zorder>toolButtonErase</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
